src/app/components/client/ExposureCard.tsx

```diff
@@ -132,48 +122,66 @@ export const ExposureCard = (props: Props) => {
-              </span>
-            </a>{" "}
-            Remove this profile to protect your privacy.
-          </p>
+          {isScanResult(props.exposureType) ? 
+          // Data broker content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-info-for-sale-part-one", 
+              {
+                elems: {"data_broker_link": <a href={exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt={l10n.getString("open-in-new-tab-alt")} width="13" height="13" aria-label={l10n.getString("open-in-new-tab-aria-label")} />
+                </span>
+              </a>
+              {" "}{l10n.getString("exposure-card-description-info-for-sale-part-two")}
+            </p>
+          </div> : 
+          // Data breach content
+          <div>
+            <p>
+              {l10n.getFragment("exposure-card-description-data-breach-part-one", 
+              {
+                vars: {
+                  data_breach_company: exposureName,
+                  data_breach_date: props.exposureType.BreachDate
+                },
+                elems: {"data_breach_link": <a href={exposureDetailsLink}/>}
+              })}
+              <a href={exposureDetailsLink}>
+                <span>
+                  <OpenInNew alt={l10n.getString("open-in-new-tab-alt")} width="13" height="13" aria-label={l10n.getString("open-in-new-tab-aria-label")} />
+                </span>
+              </a>
+              {" "}{l10n.getString("exposure-card-description-data-breach-part-two")}
+            </p>
+          </div>
+          }
           <div className={styles.exposureListOfExposureTypes}>
-            <ul>
-              <li>Your exposed info:</li>
-              <li>
-                <DetailsFoundItem
-                  icon={<MultipleUsers alt="" width="15" height="15" />}
-                  whichExposed="family"
-                  num={0}
-                />
-              </li>
-              <li>
-                <DetailsFoundItem
-                  icon={<PhoneIcon alt="" width="15" height="15" />}
-                  whichExposed="phone"
-                  num={5}
-                />
-              </li>
-              <li>
-                <DetailsFoundItem
-                  icon={<EmailIcon alt="" width="15" height="15" />}
-                  whichExposed="email"
-                  num={4}
-                />
-              </li>
-              <li>
-                <DetailsFoundItem
-                  icon={<LocationPin alt="" width="15" height="15" />}
-                  whichExposed="address"
-                  num={0}
-                />
-              </li>
-            </ul>
+            <dl>
+              <div>{l10n.getString("exposure-card-your-exposed-info")}:</div>
+              <DetailsFoundItem
+                icon={<MultipleUsers alt="" width="13" height="13" />}
```

Comment:
Not sure if this is even worth of a nit or not, but not sure if we should rename anything for consistency:

- `<MultipleUsers />`
- `<PhoneIcon />`
- `<EmailIcon />`
- `<LocationPin />`

Not sure if first and last should have `Icon` suffixes or not.